drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -6748,19 +6748,19 @@ RID RenderingDeviceVulkan::render_pipeline_create(RID p_shader, FramebufferForma
 	VkPipelineFragmentShadingRateStateCreateInfoKHR vrs_create_info;
 	if (context->get_vrs_capabilities().attachment_vrs_supported) {
 		// If VRS is used, this defines how the different VRS types are combined.
-		// combinerOps[0] decides how we use the output of pipeline and primitive (drawcall) VRS
-		// combinerOps[1] decides how we use the output of combinerOps[0] and our attachment VRS
+		// CombinerOps[0] decides how we use the output of pipeline and primitive (drawcall) VRS.
+		// CombinerOps[1] decides how we use the output of combinerOps[0] and our attachment VRS.
```

Comment:
This looks like a piece of code, so it must match the casing of that code.

Reply:
Fixed!